cli/package.json

```diff
@@ -28,6 +28,7 @@
   ],
   "dependencies": {
     "@iarna/toml": "^2.2.3",
+    "chalk": "^2.4.2",
```

Comment:
I'd suggest https://www.npmjs.com/package/kleur instead `chalk`, it has better performance and same API.